src/datagen/adequate_datagen.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @module datagen/adequate_datagen */
+/**
+ * @fileoverview Contains functions which can generate random data,
+ * random request body, random headers that complies with schema.
+ */
+
+const RandExp = require('randexp');
```

Comment:
Can you check the naming style once?

Reply:
@siriscac 
" return new RandExp(regex).gen(); "
RandExp is a class .And we invoke its constructor.
That's why we are following the naming convention of class .